webapp/src/pages/tagConfig/components/TagConfigComponent.js

```diff
@@ -185,10 +205,7 @@ class TagConfigComponent extends Component {
             <td>{tag.tag_type}</td>
             <td>
               <button
-                onClick={() => {
-                  tag.active = false; 
-                  this.onClickUpdate(tag)
-                }}>
+                onClick={this.onClickDelete(tag)}>
```

Comment:
This is what's causing your maximum update depth exceeded error. You're calling the event handler here instead of passing a function to onClick, I guess you accidentally forgot to make it a lambda:
`onClick={()=>this.onClickDelete(tag)}`